ios/Sources/SentryCapacitorPlugin/SentryCapacitorPlugin.swift

```diff
@@ -1,13 +1,32 @@
 import Foundation
 import Capacitor
 import Sentry
+import Sentry._Hybrid
```

Comment:
Lets add this option optional since it is only required when users use SPM.

```suggestion

// Keep compatibility with CocoaPods.
#if SWIFT_PACKAGE
import Sentry._Hybrid
#endif
```

Reply:
Good point 👍